pytest_plugins/fixture_markers.py

```diff
@@ -26,17 +26,15 @@ def pytest_generate_tests(metafunc):
             match_params.extend(
                 [
                     setting_rhel_ver
-                    for setting_rhel_ver in settings.content_host.rhel_versions
+                    for setting_rhel_ver in settings.supportability.content_hosts.rhel.versions
                     if re.fullmatch(str(matcher[0]), str(setting_rhel_ver))
                 ]
             )
         rhel_params = []
         filtered_versions = set(list_params + match_params)
-        for ver, data in settings.content_host.rhel_versions.items():
+        for ver in settings.supportability.content_hosts.rhel.versions:
             if ver in (filtered_versions or settings.content_host.rhel_versions):
-                # prefer version-specific deploy workflow before using the default one
-                workflow = data.get('deploy_workflow', settings.content_host.deploy_workflow)
-                rhel_params.append(dict(workflow=workflow, rhel_version=ver))
+                rhel_params.append(dict(rhel_version=ver))
```

Comment:
Where does `rhel_version` here is coming from, I dont see any Var declared with it.

OR I am missing something :)

Reply:
rhel_version isn't a variable label, its the kwarg to `dict` instantiation.

Reply:
Ysah but then Isnt it should be string, or again I m missing something.